packages/dds/tree/src/test/feature-libraries/sequence-field/rebase.spec.ts

```diff
@@ -352,4 +377,62 @@ describe("SequenceField - Rebase", () => {
         ];
         assert.deepEqual(actual, expected);
     });
+
+    it("concurrent inserts ↷ delete", () => {
+        const delA = tagChange(createDeleteChangeset(0, 1), brand(1));
+        const insertB = tagChange(createInsertChangeset(0, 1), brand(2));
+        const insertC = tagChange(createInsertChangeset(1, 1), brand(3));
+        const insertB2 = rebaseTagged(insertB, delA);
+        const insertC2 = rebaseTagged(insertC, delA, insertB2);
+        const expected = createInsertChangeset(1, 1);
+        assertDeltaEqual(insertC2.change, expected);
+    });
+
+    it("concurrent inserts ↷ connected delete", () => {
+        const delA = tagChange(createDeleteChangeset(0, 1), brand(1));
+        const delB = tagChange(createDeleteChangeset(1, 1), brand(2));
+        const delC = tagChange(createDeleteChangeset(0, 1), brand(3));
+
+        const insertD = tagChange(createInsertChangeset(0, 1), brand(2));
+        const insertE = tagChange(createInsertChangeset(3, 1), brand(3));
+        const insertD2 = rebaseTagged(insertD, delA, delB, delC);
+        const insertE2 = rebaseTagged(insertE, delA, delB, delC, insertD2);
+        const expected = createInsertChangeset(1, 1);
+        assertDeltaEqual(insertE2.change, expected);
+    });
 });
+
+function assertDeltaEqual(actual: TestChangeset, expected: TestChangeset) {
+    assert.deepEqual(toDelta(actual), toDelta(expected));
```

Comment:
Deep equality on delta will fail in some cases when it shouldn't. This is not a problem here if the tests are passing but we should avoid using it to prevent it from propagating though copy-paste. Use `assertDeltaEqual` from `src\test\utils.ts` instead.